dev/mxcube3/mxcube3-docker-debian9-dev/debian-install.sh

```diff
@@ -49,18 +49,15 @@ install_debian_deps() {
   #ffmpeg
   command apt-get -y install ffmpeg
 
-  #pillow
-  command apt-get -y install libxml2-dev libxslt-dev libtiff-dev libjpeg-dev zlib1g-dev libfreetype6-dev liblcms2-dev libwebp-dev tcl8.5-dev tk8.5-dev python-tk
-
   # npm package imagemin
   command apt-get -y install nodejs nodejs-legacy build-essential nasm libpng12-dev libpng-dev libpng++-dev libpng-tools libpng16-16 zlibc pkg-config
   command apt-get -y install libcairo2-dev libjpeg-dev libpango1.0-dev libgif-dev build-essential librsvg2-dev
-  
+
 }
 
 install_python_deps() {
-  command pip2 install numpy
-  command pip2 install -r requirements.txt
+  command pip3 install numpy
+  command pip3 install -r requirements-py3.txt
```

Comment:
With the latest master of MXCuBE3 the requirements-py3.txt have been renamed to simply requirements.txt

Reply:
Nice! I will fix it then.